b/Boost/Boost-1.78.0-GCC-12.2.0-Clang-15.0.5.eb

```diff
@@ -0,0 +1,43 @@
+##
+# Authors::   Denis Kristak <thenis@inuits.eu>
+##
+name = 'Boost'
+version = '1.78.0'
```

Comment:
In the 2022b generation (which includes easyconfigs using GCC(core) 12.2.0 as toolchain), we've settled on Boost 1.81.0.

I suspect you have a reason for using an older version here?